lightning_pose/train.py

```diff
@@ -187,6 +187,13 @@ def train(cfg: DictConfig) -> None:
     else:
         data_module_pred = data_module
 
+    model = load_model_from_checkpoint(
```

Comment:
is the goal here to ensure that the "model" object floating around has the correct weights loaded, rather than relying on proper use of "best_ckpt" below? If so you should also remove "best_ckpt" arg from L250 `export_predictions_and_labeled_video` (since otherwise we'll waste a bit of time loading the best weights into the model that already has the best weights loaded)